src/m365/aad/commands/app/app-add.ts

```diff
@@ -89,7 +91,8 @@ class AadAppAddCommand extends GraphItemsListCommand<ServicePrincipalInfo> {
       .then((_appInfo: AppInfo): void => {
         const appInfo: any = {
           appId: _appInfo.appId,
-          objectId: _appInfo.id
+          objectId: _appInfo.id,
+          tenantId: Utils.getTenantIdFromAccessToken(auth.service.accessTokens[auth.defaultResource].value)
```

Comment:
For now this approach is good. If in the future we allow users to work on multiple directories, we'll need to adjust this.